go/tools/bazel_testing/bazel_testing.go

```diff
@@ -206,15 +206,16 @@ func RunBazel(args ...string) error {
 // If the command starts but exits with a non-zero status, a *StderrExitError
 // will be returned which wraps the original *exec.ExitError.
 func BazelOutput(args ...string) ([]byte, error) {
-	return BazelOutputWithInput(nil, args...)
+	stdout, _, err := BazelOutputWithInput(nil, args...)
+	return stdout, err
 }
 
 // BazelOutputWithInput invokes a bazel command with a list of arguments and
 // an input stream and returns the content of stdout.
 //
 // If the command starts but exits with a non-zero status, a *StderrExitError
 // will be returned which wraps the original *exec.ExitError.
-func BazelOutputWithInput(stdin io.Reader, args ...string) ([]byte, error) {
+func BazelOutputWithInput(stdin io.Reader, args ...string) ([]byte, []byte, error) {
```

Comment:
I can see this being a useful change, but it looks like it is unused by this PR. Is that correct?

Reply:
Correct, I did use it while debugging why the test was failing, but ended up not needing to use it to actually validate the change. I may add an "after" handler that outputs this in case of a failure.